### PR TITLE
added aria attributes to make explore-cc button accessible

### LINKED ISSUE #727: ARIA attributes for global header
## Description

**'Explore CC'** button in the global header needs to have the following ARIA attributes to be accessible: 
- 'aria-haspopup' set to `true` to show that it opens up a menu when clicked
- 'aria-expanded' set to `false` when the global header menu is collapsed, and to `true` when it is expanded.
- its role should be set by 'aria-role: button'. Or, even better for accessibility, it should be a button so that we can get keyboard interaction and aria-roles for free. This will, however, require more effort in styling the element.

## Resolution

- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by rochisha0**
## Fixes

Fixes #727 by @obulat 

## Description

Added Aria label to bake Explore-CC accessible

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: packages/vocabulary/src/scripts/global_header.js [hunk 1]
```diff
@@ -106,6 +106,9 @@ class GlobalHeader {
           document.createTextNode(NAVIGATION_TAB_TEXT)
         ], element => {
           element.setAttribute('href', '#')
+          element.setAttribute('aria-haspopup', 'true')
+          element.setAttribute('aria-expanded', 'false')
```
Comment (dhruvkb) [inline packages/vocabulary/src/scripts/global_header.js:110]: Shouldn't we set this value to `true` or `false` based on whether the tab is opened or closed?

Comment (obulat) [reply packages/vocabulary/src/scripts/global_header.js:110]: The `aria-expanded` should be set to `false` when the tab is closed, and to `true` when the tab is open, in javascript.

**Comment by zackkrida** [comment]
We will wait on merging this until the @obulat and @dhruvkb's comments are resolved and #697 is merged.

**Comment by zackkrida** [comment]
Thanks @rochisha0 I added the aria-expanded toggling and merged the PR. Nice work!

**Pull request merged**